bitcoin/src/bip152.rs

```diff
@@ -244,8 +244,10 @@ impl HeaderAndShortIds {
                 });
             } else {
                 match version {
-                    1 => short_ids.push(ShortId::with_siphash_keys(&tx.compute_txid(), siphash_keys)),
-                    2 => short_ids.push(ShortId::with_siphash_keys(&tx.compute_wtxid(), siphash_keys)),
+                    1 =>
+                        short_ids.push(ShortId::with_siphash_keys(&tx.compute_txid(), siphash_keys)),
+                    2 => short_ids
+                        .push(ShortId::with_siphash_keys(&tx.compute_wtxid(), siphash_keys)),
```

Comment:
FTR IMO this is harder to read and the previous was better. If it had curly braces at least it'd look OK.

Reply:
Agreed, but shitty formatting is the price we pay to have consistent formatting in Rust.

Reply:
I'm still not convinced consistency is a goal worth pursuing. Although if there was a formatter that was flexible enough and a semantic merge tool we could get rid off of all these discussions and save a ton of time merging things.

Reply:
Perhaps we should be discussing `rustfmt` at the summit (maybe over drinks since I doubt there is a good solution).

Reply:
If you can teleport me there and back then we can.

Reply:
I'm already using my teleport machine for myself, its rubbish anyways - takes 14 hours to work and there are 300 other people on it.